pages/print_labels/control_plates.vue

```diff
@@ -3,7 +3,7 @@
     <b-row>
       <b-col>
         <PrintLabelsRouter />
-        <h1>Print Control plate labels</h1>
+        <h1>Print control plate labels</h1>
```

Comment:
A Control plate is a name of a distinct plate type in the lab and can stay capitalised.